app/actions/ActionTypes.ts

```diff
@@ -111,6 +111,16 @@ export const Joblistings = {
   DELETE: generateStatuses('Joblistings.DELETE') as AAT,
 };
 
+/**
+ *
+ */
```

Comment:
?

Reply:
hahaha, all the others had it so I figured I'd follow convention